full-service/src/bin/main.rs

```diff
@@ -102,7 +102,7 @@ fn consensus_backed_full_service(
     // Verifier
     let mut mr_signer_verifier =
         MrSignerVerifier::from(mc_consensus_enclave_measurement::sigstruct());
-    mr_signer_verifier.allow_hardening_advisory("INTEL-SA-00334");
+    mr_signer_verifier.allow_hardening_advisories(&["INTEL-SA-00334", "INTEL-SA-00615"]);
```

Comment:
Optional/Equivalent: Use the corresponding constant.
```suggestion
    mr_signer_verifier.allow_hardening_advisories(mc_consensus_enclave_measurement::HARDENING_ADVISORIES);
```

Reply:
Good suggestion.